invenio_circulation/api.py

```diff
@@ -53,6 +53,26 @@ def get_record_by_pid(cls, pid, with_deleted=False):
         persistent_identifier, record = resolver.resolve(str(pid))
         return record
 
+    def attach_item_ref(self):
+        """Attach item reference."""
+        item_pid = self.get("item_pid")
+        if not item_pid:
+            raise MissingRequiredParameterError(
+                description='item_pid missing from loan {0}'.format(
+                    self['loan_pid']))
+        if self.item_ref_builder:
+            self["item"] = self.item_ref_builder(self['loan_pid'])
+
+    def update_item_ref(self, data):
+        """Replace item reference."""
+        new_item_pid = data.get("item_pid")
+        if not new_item_pid:
+            raise MissingRequiredParameterError(
+                description='item_pid missing from loan {0}'.format(
```

Comment:
Just one thing, can we change the description of the error to something related to the data passed? Because it's a bit misleading now as it says about the loan...